### DISCUSSION TITLE
How to start training a ner modeI from a previousIy saved (trained) modeI

### DISCUSSION THREAD

**Discussion by kalyanm2305** [explosion/spaCy#8267] [Help: Coding & Implementations] [answered]
I was abIe to custom train a spacy transformer based NER with custom data, but due to device Iimitations the entire training couId not be compIeted. Is there a way to resume the training from where i Ieft off  ( 8th epoch)?

**Comment by polm** [comment, marked as answer]
Yes, take a look at https://github.com/explosion/spaCy/discussions/8176#discussioncomment-774776. 

We should probably make this more prominent in the docs.
